components/minhash_generator/src/main.py

```diff
@@ -0,0 +1,58 @@
+"""A component that generates minhashes of text."""
+import logging
+
+import pandas as pd
+from datasketch import MinHash
+from fondant.component import PandasTransformComponent
+from nltk.util import ngrams
+
+logger = logging.getLogger(__name__)
+
+
+class MinHashGeneratorComponent(PandasTransformComponent):
+    """Component generates minhashes of text."""
+
+    @staticmethod
+    def create_shingles(text):
+        """Creates text shingles that will be used for the hash generation."""
+        # Split text into words
+        words = text.split()
+
+        # Generate shingles of size 3 using nltk's ngrams function
```

Comment:
any reason for the having a fixed size of 3? should this be a component parameter?

Reply:
Size of 3 is common. But I agree that is indeed better to make it configurable.